elements/pl-symbolic-input/pl-symbolic-input.py

```diff
@@ -130,10 +142,13 @@ def render(element_html, data):
             html = chevron.render(f, html_params).strip()
 
     elif data['panel'] == 'answer':
-        a_tru = pl.from_json(data['correct_answers'].get(name, None))
+        a_tru = data['correct_answers'].get(name, None)
         if a_tru is not None:
             if isinstance(a_tru, str):
-                a_tru = phs.convert_string_to_sympy(a_tru, variables)
+                # this is so instructors can specify the true answer simply as a string
```

Comment:
Want to give an example of correct-answer-as-string in the docs, if we don't have one already?

Reply:
done